source/_integrations/counter.markdown

```diff
@@ -52,7 +52,7 @@ counter:
       type: integer
       default: 0
     restore:
-      description: Try to restore the last known value when Home Assistant starts.
+      description: Try to restore the last known value when Home Assistant starts. If this is true, future changes to `maximum`, `minimum`, `initial` and `step` can only be made using the `counter.configure` service.
```

Comment:
To be honest, that sounds like a bug?

Have you raised an issue to verify this wasn't a bug but by design?